src/index.tsx

```diff
@@ -1,10 +1,11 @@
-import PropTypes from 'prop-types'
-import React, { Component } from 'react'
+import * as PropTypes from 'prop-types'
+import { CanvasHTMLAttributes, Component } from 'react'
+import { Options as SignaturePadOptions } from 'signature_pad'
 import SignaturePad from 'signature_pad'
```

Comment:
```suggestion
import SignaturePad, { Options as SignaturePadOptions } from 'signature_pad'
```

I would prefer to keep imports of the same library on the same line (and generally avoid stylistic changes)